### PR TITLE
Add CONTRIBUTING.md and improve readme

### LINKED ISSUE #370: [META] Create CONTRIBUTING doc for this repo
Add CONTRIBUTING.md doc with contributor guidelines.

### PR DESCRIPTION
**Pull request by sohkai**
Fixes #370.

Also adds few notes on the branching strategy, as well as some developing and contributing notes.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+1. Fork and clone a local copy of the repository.
+1. Make the appropriate changes for the issue you are trying to address or the feature that you want to add.
+1. Push the changes to the remote repository.
+1. Submit a pull request in Github, explaining any changes and further questions you may have.
+1. Wait for the pull request to be reviewed.
+1. Make changes to the pull request if the maintainer recommends them.
+1. Celebrate your success after your pull request is merged!
+
+It's OK if your pull request is not perfect (no pull request is).
+The reviewer will be able to help you fix any problems and improve it!
```
Comment (bpierre) [inline CONTRIBUTING.md]: Nice :ok_hand:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+You can also edit a page directly through your browser by clicking the "EDIT" link in the top-right corner of any page and then clicking the pencil icon in the github copy of the page.
+
+## Styleguide and development processes
+
+For the frontends, we use [prettier](https://prettier.io/) and [eslint](https://eslint.org/) to automatically lint and format the project.
```
Comment (bpierre) [inline CONTRIBUTING.md]: Maybe we could add that we generally have a `npm run lint` task to run ESLint with the right configuration, and that we generally provide a `.eslintrc` and a `.prettierrc` file (not strictly the case everywhere, but it’s easy to fix)?

File: README.md [hunk 3]
```diff
@@ -23,15 +26,41 @@ The following apps are still under development, not ready for production deploym
...
+
+Running tests of an individual app can be done by running `npm run test` inside an individual app's directory, or through the selective `npm run test:<app>` scripts.
+
+By default, tests are run in a in-memory instance of testrpc.
+
+### App frontends
+
+Developing the frontends locally can be a bit tricky, due to the frontends' reliance on a properly configured Aragon client.
+
+For now, you can follow [@bpierre's gist](https://gist.github.com/bpierre/22572415b90577aa17e5953fa68980eb#developing-the-apps) to start an app's local frontend server and connect it to a locally running Aragon client.
```
Comment (bpierre) [inline README.md]: Should we start to move this in every frontend-related repository, with a name like `CONTRIBUTING_FRONTEND.md` or `QUICKSTART_FRONTEND.md`, even if it’s still incomplete?

File: CONTRIBUTING.md (hunk 2 above)
Comment (sohkai) [reply CONTRIBUTING.md]: Done in 499d93c.

File: README.md (hunk 3 above)
Comment (sohkai) [reply README.md]: I think we can keep in the CONTRIBUTING and point people to it, perhaps if it's still hard to find we can move it to its own file :smile:.

I've also added it in aragon/aragon: https://github.com/aragon/aragon/pull/880.

**Pull request merged**